src/sentry/models/event.py

```diff
@@ -412,26 +409,6 @@ def __init__(self, snuba_values):
         )
         self.data = NodeData(node_id, data=None, wrapper=EventDict)
 
-    def __getattr__(self, name):
-        """
-        Depending on what snuba data this event was initialized with, we may
-        have the data available to return, or we may have to look in the
-        `data` dict (which would force a nodestore load). All unresolved
-        self.foo type accesses will come through here.
-        """
-        if name in ("_project_cache", "_group_cache", "_environment_cache"):
-            raise AttributeError()
-
-        allowed_attributes = ["timestamp", "event_id", "group_id", "project_id"]
-
-        if name not in allowed_attributes:
-            logger.warn("event.invalid-attribute", extra={"attribute_name": name})
```

Comment:
Did you check this is never being logged today ?

Reply:
Yes, the only thing getting logged now is `__getstate__` and `__getnewargs__`. I'm not sure how this could've worked before or why nothing appeared to be broken.